src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -1,12 +1,25 @@
 package core.basesyntax;
 
+import java.time.DateTimeException;
 import java.time.LocalDate;
 import java.time.LocalDateTime;
 import java.time.LocalTime;
 import java.time.OffsetDateTime;
+import java.time.ZoneOffset;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.Locale;
 import java.util.Optional;
 
 public class JavaDateTimeApi {
+
+    private static final String UKRAINE_OFFSET = "+02:00";
+    private static final DateTimeFormatter MONTH_FORMATTER
+            = DateTimeFormatter.ofPattern("d MMM yyyy", Locale.ENGLISH);
+    private static final DateTimeFormatter OUT_FORMATTER
```

Comment:
```suggestion
    private static final DateTimeFormatter DATE_TIME_FORMATTER
```
you may change this to another name, let's just make it clearer